includes/class-wc-gateway-ppec-plugin.php

```diff
@@ -152,6 +152,8 @@ public function bootstrap() {
 				throw new Exception( __( '%s in WooCommerce Gateway PayPal Express Checkout plugin can only be called once', 'woocommerce-gateway-paypal-express-checkout' ), self::ALREADY_BOOTSTRAPED );
 			}
 
+			load_plugin_textdomain( 'woocommerce-gateway-paypal-express-checkout', FALSE, basename( dirname( __FILE__ ) ) . '/languages/' );
```

Comment:
Better to call this during `init` as `plugins_loaded` is too early.  So I suggest to add:

~~~php
add_action( 'init', array( $this, 'load_plugin_textdomain' ) );
~~~

in `maybe_run`.

Also we've `$this->plugin_path` that stores plugin's path. So the call to `load_plugin_textdomain` can be adjusted to:

~~~php
load_plugin_textdomain( 'woocommerce-gateway-paypal-express-checkout', false, plugin_basename( $this->plugin_path ) . '/languages' );
~~~